include/cantera/numerics/CVodesIntegrator.h

```
@@ -66,6 +66,28 @@ class CVodesIntegrator : public Integrator
     }
     virtual double sensitivity(size_t k, size_t p);
 
+    //! Use this function to get nonlinear solver stats from cvodes
+    //! @param stats a long int pointer with at least two spaces
+    virtual void getNonlinSolvStats(long int* stats) const;
+
+    //! Use this function to get linear solver stats from cvodes
+    //! @param stats a long int pointer with at least eight spaces
+    virtual void getLinSolvStats(long int* stats) const;
+
+    //! Use this function to throw an error if preconditioning is specified with an
+    //! invalid problem type
+    virtual void preconditionerError();
```

I don't think this function is necessary -- you can just throw an exception directly from the locations where this problem is checked.